### ISSUE TITLE
HSL has decimal places

### ISSUE THREAD

**Issue by smlombardi** [mfouquet/Color-Copier#9]
A lot of them!

Should do a match.floor() on the numbers.

**Comment by rizwanjavaid** [comment]
> A lot of them!
> 
> Should do a match.floor() on the numbers.

Yes, please. Being able to choose the precision would be very helpful. Thanks.
